templates/solutions/_form-big-data-ai.html

```diff
@@ -45,7 +45,7 @@ <h2 class="p-heading--1">Get in touch with the Data & AI team</h2>
                   <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
                 </label>
               </div>
-            <div class="p-section--shallow">By submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</div>
+            <div class="p-section--shallow">By submitting this form, I confirm that I have read and agree to <a href="https://www.ubuntu.com/legal/dataprivacy">Canonical's Privacy Policy</a>.</div>
```

Comment:
shouldn't we preserve a link to Privacy Notice page as well and redirect people to https://ubuntu.com/legal/data-privacy/contact?

Reply:
This matches what's on [the doc](https://docs.google.com/document/d/1ZDYZHdATF7Pqlmkq9s-ucN9YdCu7ZA648vlGkyQFApc/edit) for this form